sdk/storage/Azure.Storage.Blobs/tests/BlobTestBase.cs

```diff
@@ -572,5 +573,36 @@ public void Handle(BlobQueryError blobQueryError)
                 Assert.AreEqual(_expectedBlobQueryError.Position, blobQueryError.Position);
             }
         }
+
+        /// <summary>
+        /// Gets a custom account SAS where the permissions, services and resourceType
+        /// comes back in the string character order that the user inputs it as.
+        /// </summary>
+        /// <param name="permissions"></param>
+        /// <param name="services"></param>
+        /// <param name="resourceType"></param>
+        /// <param name="sharedKeyCredential"></param>
+        /// <returns></returns>
+        public string GetCustomAccountSas(
```

Comment:
I can update the comment to something like add
```
        /// e.g. If someone has the services (srt) value as "ocs", then in the
        /// sas token returned by this method will have the srt=osc and the
        /// string-to-sign will be signed with the services value as osc.
        ///
        /// This is custom because using the regular AccountSasBuilder does not accept
        /// a raw string services or resourceTypes, it accepts services and resourceTypes
        /// (e.g. see AccountSasPermissions, AccountSasServices, AccountSasResourceType).
        /// The order is fixed respective to what values is set.
```

Reply:
So as per the other comment, I moved this method to sdk/storage/Azure.Storage.Common/tests/Shared/StorageTestBase.SasVersion.cs
Updated the comment there.